.travis.yml

```diff
@@ -22,8 +22,6 @@ env:
   matrix:
   - INSTANCE=default-centos-6
   - INSTANCE=default-centos-7
-  - INSTANCE=default-debian-8
-  - INSTANCE=default-debian-9
```

Comment:
these should be called out in the `CHANGELOG` as a `### Breaking Changes`

Reply:
Good call, updated.